pyston/utils/__init__.py

```diff
@@ -343,18 +340,22 @@ def get(self, key):
     def append(self, field):
         if isinstance(field, RESTField):
             rest_field = field
+        elif isinstance(field, six.string_types):
+            rest_field = self._create_field_from_string(field)
+        elif isinstance(field, (list, tuple)):
+            rest_field = self._create_field_from_list(field)
         else:
-            rest_field = RESTField(field)
+            raise ValueError('field can be only list, tuple or string ({} [{}])'.format(field, type(field)))
 
         if rest_field.name in self.fields_map:
-            rest_field = rest_field.join(self.fields_map[rest_field.name])
+            rest_field = self.fields_map[rest_field.name].join(rest_field)
```

Comment:
při přidání prvku do fieldsetu se dal na první místo ale já ho chci na poslední, je to teoreticky jedno nicméně toto je správnější